.agents/AGENTS.md

```
@@ -0,0 +1,70 @@
+timestamp: "2026-08-22T12:00:00Z"
+topics: ["dsom", "documentation"]
+description: "OKF-compliant documentation for AGENTS.md."
+resource: "file:///.agents/AGENTS.md"
+---
+# The Core AI Rulebook (DSOM)
+
+> **Entry Point 2:** This document is the Cognitive Entry Point (AI Persona & Rules). See [START-HERE](https://linuxmalaysia.github.io/deep-state-of-mind-for-my-ai/START-HERE/) for the master onboarding roadmap.
+
+Welcome to the Sovereign AI Agent Workspace. You are a Cognitive Digital Twin operating on the Deep State of Mind (DSOM) framework.
+
+## Core Rules:
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)